bin/removeNPMAbsolutePaths

```diff
@@ -16,11 +16,30 @@ const opts = {
   force: false,
 };
 
-args.slice(1).forEach((arg) => {
-  if (arg === '--force') {
-    opts.force = true;
+let argWarning = [];
+
+for (let i = 0; i < args.length; i += 1) {
+  const arg = args[i];
+  switch (arg) {
+    case '--force':
+      opts.force = true;
+      break;
+    case '--fields':
+      opts.fields = opts.fields || [];
+      while (args[i + 1] && args[i + 1].slice(0, 2) !== '--') {
+        opts.fields.push(args[i += 1]);
+      }
+      break;
+    default:
+      argWarning = ['Invalid argument.', 'Extra arguments are being ignored'];
+      break;
   }
-});
+}
+
+if (argWarning.length) {
+  console.warn.apply(null, argWarning);
```

Comment:
We can print the list with a nice message as:
`console.warn(`The following options are unknown and will be ignored:\n${ignoredOptions.join('\n')}`)`